### PR TITLE
feat: adapt license for poc

### PR DESCRIPTION
**Pull request by ElonMuskkkkkk**
### Summary
:sparkles::wrench::hammer:

Added support for free license feature in sealos. This involved removing license expiration check, moving license recharge logic, creating admin user and free license resources, and applying them to the cluster.

> _`admin-user`, `license`_
> _New YAML manifests for Sealos_
> _Free in the springtime_

### Walkthrough
* Remove license expiration check logic from `Reconcile` function of `LicenseReconciler` struct ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-252532f462fce5f86a27a9c8d6327465943d4e1dc49c5323bf2ecc139fbf9adeL22),[link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-252532f462fce5f86a27a9c8d6327465943d4e1dc49c5323bf2ecc139fbf9adeL138-R138))
* Move license recharge logic to execute after verifying license signature in `Reconcile` function of `LicenseReconciler` struct ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-252532f462fce5f86a27a9c8d6327465943d4e1dc49c5323bf2ecc139fbf9adeL138-R138))
* Add empty line to `controller` package for readability and formatting ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-252532f462fce5f86a27a9c8d6327465943d4e1dc49c5323bf2ecc139fbf9adeR130))
* Add `User` resource named `admin` with custom CSR expiration time to `admin-user.yaml` file in `deploy/cloud/manifests` directory ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-45b52180320b999cc5bbbc606f8006e3ad65267ea79365b83bf17e93993c4a4bL1-R5))
* Add `License` resource named `license` with token containing free license for admin user to `free-license.yaml` file in `deploy/cloud/manifests` directory ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-a57c237d3aa854c2219ff7f43f57881847aa2d1993b04a40b4741516bedcaa9eL1-R6))
* Apply `admin-user.yaml` and `free-license.yaml` manifests to Kubernetes cluster in `init.sh` script in `deploy/cloud/scripts` directory ([link](https://github.com/labring/sealos/pull/3594/files?diff=unified&w=0#diff-7f4df0129665fe81f0d8ddc6a9375d92c5062f99482ec60a9b3d4f401e667844R127-R134))

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ on:
       - "service/**"
       - "webhooks/**"
       - "frontend/**"
+      - "deploy/cloud/manifests/free-license.yaml"
```
Comment (lingdie) [inline .github/workflows/ci.yml:27]: why add this?

File: controllers/licenseissuer/deploy/manifests/deploy.yaml [hunk 2]
```diff
@@ -609,7 +609,7 @@ spec:
             secretKeyRef:
               key: isMonitor
               name: licenseissuer-env
-        image: ghcr.io/labring/sealos-licenseissuer-controller:latest
+        image: registry.cn-hangzhou.aliyuncs.com/fckc/cloud-controller:v3.3.5
```
Comment (lingdie) [inline controllers/licenseissuer/deploy/manifests/deploy.yaml:612]: do not use personal registry.

File: deploy/cloud/scripts/init.sh [hunk 3]
```diff
@@ -56,6 +56,22 @@ function sealos_run_controller {
...
+function sealos_authorize {
+  echo "start to authorize sealos"
+
+  echo "create admin-user"
+  # create admin-user
+  kubectl apply -f manifests/admin-user.yaml
+  
+  # wait for admin-user ready
+  echo "waiting for admin-user generated"
+  sleep 3
```
Comment (lingdie) [inline deploy/cloud/scripts/init.sh:68]: use `while` to make sure.

File: deploy/cloud/manifests/free-license.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+apiVersion: infostream.sealos.io/v1
```
Comment (bxy4543) [inline deploy/cloud/manifests/free-license.yaml:1]: add `// nosemgrep: generic.secrets.security.detected-jwt-token.detected-jwt-token` to ingore semgrep ci .

**Review by lingdie — APPROVED** [review]

**Pull request merged**